go.mod

```diff
@@ -1,19 +1,35 @@
 module github.com/ltcsuite/ltcwallet
 
 require (
-	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/btcsuite/golangcrypto v0.0.0-20150304025918-53f62d9b43e8
+	github.com/ltcsuite/ltcd v0.20.1-beta
+	github.com/ltcsuite/ltclog v0.0.0-20160817181405-73889fb79bd6
+	github.com/ltcsuite/ltcutil v0.0.0-20190425235716-9e5f4b9a998d
+	github.com/ltcsuite/ltcwallet/wallet/txauthor v1.0.0
+	github.com/ltcsuite/ltcwallet/wallet/txrules v1.0.0
+	github.com/ltcsuite/ltcwallet/walletdb v1.2.0
+	github.com/ltcsuite/ltcwallet/wtxmgr v1.0.0
 	github.com/btcsuite/websocket v0.0.0-20150119174127-31079b680792
-	github.com/coreos/bbolt v1.3.2
 	github.com/davecgh/go-spew v1.1.1
 	github.com/golang/protobuf v1.2.0
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/jrick/logrotate v1.0.0
-	github.com/lightninglabs/gozmq v0.0.0-20180324010646-462a8a753885
-	go.etcd.io/bbolt v1.3.3 // indirect
-	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
-	golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3
+	github.com/kkdai/bstream v0.0.0-20181106074824-b3251f7901ec // indirect
+	github.com/lightninglabs/gozmq v0.0.0-20191113021534-d20a764486bf
+	github.com/lightninglabs/neutrino v0.11.0
```

Comment:
we'll be using the ltcsuite/neutrino module here

Reply:
fixed with https://github.com/ltcsuite/ltcwallet/pull/4/commits/91b5386886079db1dbebac1333d4b9ccde6b21ab